IdentityCore/src/oauth2/aad_v2/MSIDAADV2WebviewFactory.m

```diff
@@ -36,16 +36,7 @@ @implementation MSIDAADV2WebviewFactory
 {
     NSMutableDictionary<NSString *, NSString *> *parameters = [super authorizationParametersFromConfiguration:configuration
                                                                                                  requestState:state];
- 
-    NSMutableOrderedSet<NSString *> *allScopes = parameters[MSID_OAUTH2_SCOPE].msidScopeSet.mutableCopy;
-    if (!allScopes)
-    {
-        allScopes = [NSMutableOrderedSet new];
-    }
-    
-    parameters[MSID_OAUTH2_SCOPE] = allScopes.msidToString;
```

Comment:
We don't send scopes in url any more?

Reply:
This is already set in MSIDWebViewFactory. I'm not sure what this code was doing to be honest.

Reply:
hmm, you are right. Now I find that i don't understand this part of code neither :P

Reply:
:)